html-css/week2/index.html

```diff
@@ -0,0 +1,107 @@
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>How the Form Builder Works</h2>
+            <p>Learn how to customize forms, welcome emails, and other response emails with our Form Builder.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>How Mailchimp Prevents and Handles Abuse</h2>
+            <p>Learn the steps we take to prevent abuse of our system, and how we handle abuse when it happens.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>How to Allowlist Delivery IP Addresses</h2>
+            <p>Learn how to enable company allowlisting to help ensure your Mailchimp campaigns get to all your subscribers.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>How Legitimate Marketers Can Prevent Spam Complaints</h2>
+            <p>Learn why audiences generate abuse complaints, and steps to take to prevent them.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>How Mailchimp Can Help Your Online Store</h2>
+            <p>Mailchimp’s e-commerce tools help you grow your business and sell more stuff. Find new audiences, promote your products, boost customer loyalty, and track your sales.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>Apple Privacy Update FAQs</h2>
+            <p>In this article, you'll get answers to common questions about Apple's new privacy protection update.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>About Authorized Applications</h2>
+            <p>Learn about authorized applications and what it means to grant access to your account information.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="results">
+            <h2>About Geolocation</h2>
+            <p>Mailchimp determines location and time zone for your subscribers by collecting data when they sign up for your marketing or interact with your email campaigns.</p>
+            <i class="fa-solid fa-angle-right"></i>
+        </section>
+        <section class="pagination">
+            <span>1 - 10 of 625 results</span>
+            <i class="fa-solid fa-angle-left icon light"></i>
+            <i class="fa-solid fa-angle-right icon"></i> 
+        </section>
+        <section class="contact-support">
+            <h3>Contact Support</h3>
+            <p>You have 24/7 access to award-winning live chat and email support with your paid plan.</p>
+            <div class="button">
+                <button id="email" onclick="#">Email Us</button>

```

Comment:
Ids are mostly used for Javascript purposes, for styling it's preferable to use classes. Here, the email button and the chat button have the same styles, so you could have used a reusable class "communication-button" instead of repeating the same style for the two ids. In software, it's always a good practice to do DRY code (Don't Repeat Yourself).